src/main/kotlin/no/nav/familie/ef/sak/repository/domain/Vilkårsvurdering.kt

```diff
@@ -53,7 +53,7 @@ enum class VilkårType(val beskrivelse: String,
     OPPHOLDSTILLATELSE("Vises kun for ikke-nordiske statsborgere - " +
                        "Foreligger det oppholdstillatelse eller annen bekreftelse på gyldig opphold?"),
     FORUTGÅENDE_MEDLEMSKAP("§15-2 Forutgående medlemskap",
-                           listOf(DelvilkårType.TRE_ÅRS_MEDLEMSKAP, DelvilkårType.DOKUMENTERT_FLYKTNINGSTATUS)),
+                           listOf(DelvilkårType.TRE_ÅRS_MEDLEMSKAP)),
```

Comment:
For "oppklaring" - rename tre_års_medlemskap til fem_års_medlemskap eller noe som er litt mer generelt?

Reply:
Ja, tenkte egentlig at det kunne være en fin ryddejobb å ta alle disse 3 -> 5 år samtidig

Reply:
👍